src/svelte/src/components/dataEditor.svelte

```diff
@@ -220,6 +220,7 @@ limitations under the License.
         (targetLine == 0 ? 0 : targetLine - 1) * lineHeight
     }
     closeEphemeralWindows()
+    clearDataDisplays()
```

Comment:
Is there any time where we'd close the ephemeral window and not clear the data display?  If not, let's just clear the data display ni the close ephemeral window function instead to make it simpler and less error prone.